includes/class-freemius.php

```diff
@@ -17789,10 +17392,29 @@ function opt_in(
                 );
 
                 $this->maybe_modify_api_curl_error_message( $result );
+                $this->maybe_modify_api_blocked_error_message( $result );
+
+                $is_connected = null;
+
+                if ( empty( $license_key ) && $this->is_enable_anonymous() ) {
+                    $this->skip_connection( fs_is_network_admin() );
+
+                    $is_connected = ( ! FS_Api::is_blocked( $result ) );
+                }
+
+                $this->store_connectivity_info(
+                    (object) array( 'is_active' => true ),

```

Comment:
@fajardoleo, please add a comment above explaining why `'is_actiove'` is `true`, which I assume is because if the user is opting in or activating a license, it means the SDK was turned on for them. Otherwise, this logic would not be executed in the first place.